lib/resty/websocket/client.lua

```diff
@@ -128,6 +128,13 @@ function _M.connect(self, uri, opts)
             end
             ssl_verify = true
         end
+
+        if opts.headers then
+          headers = opts.headers
+          if type(headers) ~= "table" or #headers > 0 then
```

Comment:
Use of `#` on Lua tables involves an expensive `lj_tab_len()` call which is an `O(n)` operation. So it's not worth the cost here.

Reply:
Should we simply remove the check?

Reply:
@salazar That works.